app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -415,6 +415,18 @@ class NewMessageFragment : Fragment() {
         binding.fromMailAddress.text = formattedExpeditor
     }
 
+    private fun doAfterSubjectChange() {
+        binding.subjectTextField.doAfterTextChanged { editable ->
+            editable?.toString()?.let { newMessageViewModel.subjectTextField = it }
+        }
+    }
+
+    private fun doAfterBodyChange() {
+        binding.bodyTextField.doAfterTextChanged { editable ->
+            editable?.toString()?.let { newMessageViewModel.bodyTextField = it }
+        }
+    }
+
```

Comment:
Can you access the content when saving instead of saving it continuously?

Reply:
No, because the Activity's `onStop()` always triggers before the Fragment's `onStop()`.
It's the one from the Activity that tells the Fragment to start its own.